### PR TITLE
Lasato ocs docs reorg

### PR DESCRIPTION
**Pull request by lasato-OSI**
Hi Rainier,

Please review this pull request.

FYI: I was unable to add comments to my toc.yml file. Doing so created an invalid toc.yml file.

DO NOT approve the pull request until tomorrow, Tuesday, 7/14/20.

### CODE REVIEW

File: Content_Portal/Documentation/DataViews/DefineDataView/Available_Field_Sets.md [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
 
-# Available Field Sets
+# Retrieve available field sets
 
 Available [field sets](xref:DataViewsFieldSets) include fields not currently included in the data view, which are available to be added. 
 
-The list of field sets is generated based on the `.Queries` and `.GroupingFields` of the defined data view. These fields and field sets can be readily added to the data view `.DataFieldSets`. Once the field is added, it is omitted from the list of available field sets. Similarly, if the field is removed from the data view, it is added back to the available field sets. This allows an iterative approach to building the data view by easily adding and removing fields from the data view.
+The list of field sets is generated based on the `.Queries`, `.GroupingFields`, and `.IndexTypeCode` of the defined data view. These fields and field sets can be readily added to the data view `.DataFieldSets`. Once the field is added, it is omitted from the list of available field sets. Similarly, if the field is removed from the data view, it is added back to the available field sets. This allows an iterative approach to building the data view by easily adding and removing fields from the data view.
```
Comment (jamesbperry) [inline Content_Portal/Documentation/DataViews/DefineDataView/Available_Field_Sets.md:9]: Why `.IndexTypeCode`?

File: Content_Portal/Documentation/DataViews/DefineDataView/Consolidate_Data_Fields.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+uid: ConsolidateDataFields
+---
+
+# Consolidate data fields
+Data views containing streams that refer to the same property by different names will return a null value for both. This typically arises from a stream used in a data view having an identical property named slightly differently than the corresponding property in the data view, such as "Temperature" vs "AmbientTemperature".
```
Comment (jamesbperry) [inline Content_Portal/Documentation/DataViews/DefineDataView/Consolidate_Data_Fields.md:6]: This is misleading

File: Content_Portal/Documentation/DataViews/DefineDataView/Add_Identifying_Field.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+uid: AddIdentifyingField
+---
+
+# Add identifying field
```
Comment (jamesbperry) [inline Content_Portal/Documentation/DataViews/DefineDataView/Add_Identifying_Field.md:5]: I'm wondering how helpful this page is by itself. Its (broken?) link is to a section with almost-identical text, but with an example step showing the effect of adding an Identifying field.

Identifying fields can be a key concept for properly shaping a data view. Perhaps this page is an opportunity to illustrate the concept via a small, standalone example?

File: Content_Portal/Documentation/Assets/Access_Control.md [hunk 4]
```diff
@@ -0,0 +1,357 @@
+
+
+
+# Access Control API
+The Access Control API is used to secure Assets by setting their ownership and permissions. For more information about OCS access control, see Role-based access control. 
+
+***
+## `Get Assets Access Control List ` 
+Get the default `AccessControlList` for the Assets collection.
+[QUESTION: Is Assets capitalized? Assets collection, not Asset collection]
```
[the remark this answers is not in the document]
Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/Access_Control.md:10]: Skip reviewing this file for now. It is only being added to GitHub.

File: Content_Portal/Documentation/Assets/Access_Control.md [hunk 5]
```diff
@@ -0,0 +1,357 @@
...
+        }
+    ]
+} 
+```
+
+***
+
+## `Update Assets Access Control List` 
+
+Update the default `AccessControlList` for the Assets collection
```
[the remark this answers is not in the document]
Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/Access_Control.md:69]: Please skip reviewing this file for now. It is only being added to GitHub.

File: Content_Portal/Documentation/DataViews/DefineDataView/Consolidate_Data_Fields.md (hunk 2 above)
Comment (jamesbperry) [reply Content_Portal/Documentation/DataViews/DefineDataView/Consolidate_Data_Fields.md:6]: How about:

# Consolidate data fields
 
Streams included in a data view may have slight differences in property naming, despite those properties representing the same logical thing. For example: data from one equipment manufacturer reports "Temperature," while another reports "AmbientTemperature" instead.
 
Data views can overcome property naming differences by consolidating these similar properties into a single data field. This applies to stream properties referenced by id (Field Source=PropertyId) or by name (Field Source=PropertyName), and to stream metadata keys (Field Source=Metadata). For those cases, adding multiple match values to the field's Keys collection will allow alternate keys to match. See [Consolidate data fields] in the Define a data view topic for details and an example.

File: Content_Portal/Documentation/DataViews/DefineDataView/Available_Field_Sets.md (hunk 1 above)
Comment (osimarkrogers) [reply Content_Portal/Documentation/DataViews/DefineDataView/Available_Field_Sets.md:9]: Hi Brandon - - we can address the points you raise right after the merge, as it is only to reflect the reorg.  All content issues will be on the table moving forward.  (Mark Rogers)

Comment (jamesbperry) [reply Content_Portal/Documentation/DataViews/DefineDataView/Available_Field_Sets.md:9]: Thanks Mark - Ok so, this version of the docs will not go live publicly until after some further revision?

**Comment by osimarkrogers** [comment]
Hi Brandon - - this will go live tomorrow, but we can immediately make revisions and push changes as needed.  I’ll shoot you a meeting for tomorrow if that works…I realize it’s the end of the sprint, so we could do Thursday if that’s preferable.

Thanks,

Mark

File: docfx.json [hunk 6]
```diff
@@ -21,9 +21,8 @@
     "content": [
       {
         "files": [
-          "Documentation/**/toc.yml",
-          "Documentation/**/**.md",
-          "Documentation/toc.yml",
+	  "Content_Portal/**/**.md",
+	  "Content_Portal/**/toc.yml",
```
Comment (rmuolic) [inline docfx.json:25]: Looks like you are missing a line:

"Content_Portal/toc.yml",

Comment (lasato-OSI) [reply docfx.json:25]: Content_Portal/**/doc.yml includes Content_Portal/toc.yml

**Pull request merged**